typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -108,11 +119,17 @@ async function main(): Promise<boolean> {
   return true;
 }
 
+interface warpRouteInfo {
+  balance: number;
+  value?: number;
```

Comment:
I could see it not being super clear that value means the USD value, some comments or a rename of `valueUsd` could help